### PR TITLE
#687 New parameter HighlightOnActiveChild + CSS

### PR DESCRIPTION
**Pull request by crdo**

### CODE REVIEW

File: Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿@namespace Havit.Blazor.Components.Web.Bootstrap
 <CascadingValue Value="this" IsFixed="true">
-    <div>
+    <div class="@CssClassHelper.Combine(HighlightOnActiveChild ? "highlight-on-active-child" : null)">
```
Comment (hakenr) [inline Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor:3]: The name of the class should start with `.hx-sidebar-item-...`

File: Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs [hunk 2]
```diff
@@ -36,6 +36,12 @@ public partial class HxSidebarItem : IAsyncDisposable
 	/// </summary>
 	[Parameter] public bool ExpandOnMatch { get; set; } = true;
 
+	/// <summary>
+	/// Set to <c>false</c> if you don't want to highlight the item if one of the children URL matches.<br/>
+	/// Default is <c>true</c>.
+	/// </summary>
+	[Parameter] public bool HighlightOnActiveChild { get; set; } = true;
```
Comment (hakenr) [inline Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs:43]: In current state, I would probably go for `false` as the default.
Most of the sidebars I found are not highlighting the parent node et al.
A few do, but usually in different (less accented) style that the item itself.
The visual I like is GitHub:
image

...consider making the visual style separately defined.
...consider "just bold" for the default parent highlighting (in such case I would vote for `true` as default value :-D)

Comment (crdo) [reply Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs:43]: All done.

**Review by hakenr — APPROVED** [review]
lgtm

**Pull request merged**